mne/epochs.py

```diff
@@ -343,13 +343,16 @@ def preload_data(self):
         assert self._data.shape[-1] == len(self.times)
         return self
 
-    def decimate(self, decim, copy=False):
+    def decimate(self, decim, offset=0, copy=False):
```

Comment:
I think we need to call it `sample_offset` (to delineate it from a time offset), and it needs to go after `copy` to avoid an API change.